.travis.yml

```diff
@@ -11,9 +11,12 @@ env:
     - IS_RELEASE=$(git ls-remote origin | grep "$TRAVIS_COMMIT\s\+refs/heads/release$")
     - RELEASE_TAG=$(git tag -l --points-at HEAD | head -n1)
     - BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
-    - DJANGO_ACCELERATOR_REVISION=`./infer_django_accelerator_revision.sh $BRANCH`
+    - DJANGO_ACCELERATOR_REVISION=`./infer_repository_revision.sh "$BRANCH" https://www.github.com/masschallenge/django-accelerator.git`
+    - MENTOR_DIRECTORY_REVISION=`./infer_repository_revision.sh "$BRANCH" https://$GITHUB_ACCESS_TOKEN@github.com/masschallenge/directory.git`
 
 before_install:
+- echo $MENTOR_DIRECTORY_REVISION
+- echo $DJANGO_ACCELERATOR_REVISION
```

Comment:
remove echo calls or add more context so its useful output.

Reply:
Used it for debugging, thought I removed it already.